accepted/0000-staged-packages.md

```diff
@@ -0,0 +1,90 @@
+# RFC: Staged Packages
+
+## Summary
+
+Staging workflow which allows CI to publish new package versions, while allowing
+humans to be the final arbiters of general availability.
+
+## Motivation
+
+Anything published by CI is instantly available for general use, which isn't
+ideal when a human is required to be in the loop. And even when fully automated
+CI/CD is allowed, the inability to use 2FA with auth tokens is a security risk.
+
+## Detailed Explanation
+
+We need a minimum of four things to solve these problems:
+
+1. Staging area on the registry invisible to most `npm` commands
+2. Auth tokens that can be scoped to the staging area
```

Comment:
In some of the conversations that happened before the initial RFC (but which I don't see reflected in the RFC here) the assumption was that either a specific token could be restricted to only doing staged publishes in general, or a user/team could be granted access to only stage in a given package.

The complexity around that part is a bit orthogonal to the rest of this, however, and so for now, I think it's best to leave it aside.  For the purpose of limiting the scope of _this_ RFC, we can assume that users who are maintainers can both stage, publish, and promote, since it's up to the server to allow or reject any given action based on ACLs and token metadata.